examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3721,8 +4050,26 @@ class GLTFParser {
 
 				if ( node === undefined ) continue;
 
-				node.updateMatrix();
-				node.matrixAutoUpdate = true;
+				const ext = target.extensions;
+				let animationPointerPropertyPath = ext ? ext[ EXTENSIONS.KHR_ANIMATION_POINTER ]?.path : null;
+				if ( animationPointerPropertyPath ) {
+
+					animationPointerPropertyPath = animationPointerPropertyPath.replaceAll( '/', '.' );
+					// replace node/material/camera/light ID by UUID
+					const parts = animationPointerPropertyPath.split( '.' );
+					parts[ 2 ] = node.uuid;
+					animationPointerPropertyPath = parts.join( '.' );
+					if ( animationPointerDebug )
+						console.log( node, inputAccessor, outputAccessor, target, animationPointerPropertyPath );
```

Comment:
The trouble with targeting animation to the UUID is that many users will clone the model before attaching one or more copies to their scene. The cloned objects will have new UUIDs, and so the same animation clips cannot target them. We prefer to target based on unique names for that reason, but glTF doesn't require unique names. And in this case (see above) we may need to animate multiple objects, even if only a single source glTF definition was animated. 😕

Reply:
Switched to `node.name ?? node.uuid`, similar to other places.